### DISCUSSION TITLE
undefined method msqli_stmt error

### DISCUSSION THREAD

**Discussion by geomodules1** [geodesicsolutions-community/geocore-community#201] [General]
If you are receiving this undefined method msqli_stmt error which is normally on GoDaddy servers

turn off mysqli 
turn on mysqlnd

Do this under php setttngs and then everything will work

**Comment by geosmarty** [comment]
As well, you can choose between using either ALT-PHP vs EA-PHP for your php server environment.  ALT-PHP includes PHP versions that CloudLinux provides along with their PHP Selector feature.  These versions of PHP are often hardened and patched against vulnerabilities to allow users to continue to use applications that require outdated PHP versions.  EA-PHP is the PHP versions that cPanel packages and provides via the EasyApache 4 repositories.   These are the standard versions that are included in a base-installation of cPanel and are managed via the MultiPHP Manager in both cPanel & WHM interface.
